product_configurator/models/product_attribute.py

```diff
@@ -45,8 +45,24 @@ def onchange_custom_type(self):
         'disable a attribute without deleting it'
     )
 
-    min_val = fields.Integer(string="Min Value", help="Minimum value allowed")
-    max_val = fields.Integer(string="Max Value", help="Minimum value allowed")
+    min_val = fields.Integer(
+        string="Min Value",
+        help="Minimum value allowed"
+    )
+    max_val = fields.Integer(
+        string="Max Value",
+        help="Maximum value allowed, or 0 if not checked"
+    )
+    min_fval = fields.Float(
+        string="Min Float Value",
+        digits=(16, 4),
+        help="Minimum value allowed"
+    )
+    max_fval = fields.Float(
+        string="Max Float Value",
+        digits=(16, 4),
+        help="Maximum value allowed, or 0 if not checked"
+    )
```

Comment:
Can't we keep one float field for min/max of int and float since there can only be one custom field type per attribute?